README.md

```diff
@@ -1,2 +1,65 @@
-# members-only
-Members only blog
+# Members Only
+
+This is a blog app which authenticates and authorizes users. If a user is not signed in, they cannot see the authors of the posts. However, once they sign in, all the author names are visisble. Users also cannot edit the post of other users; they are also restricted to editing their own posts.  
+
+![image](https://user-images.githubusercontent.com/43865875/101761969-7a608600-3ae5-11eb-9d98-26d697adbb0d.png)
+
+[Live demo link](https://tranquil-island-76594.herokuapp.com/)
+
+
+## Built With
+
+- Ruby -v  2.7.2
+- Rails -v 6.0.3.4
+- SQLite3
+- Devise
+- Bootstrap
+
+
+## Getting Started
+
+### Prerequisites
+
+To get this project up and running locally, you must already have Ruby, the necessary Gems, and Ruby on Rails installed on your computer.
+
+**To get this project set up on your local machine, follow these simple steps:**
+
+1. Open Terminal.
+2. Navigate to your desired location to download the contents of this repository.
+3. Run ```git clone https://github.com/sinansevgi/members-only.git```.
+4. Run ```cd members-only```.
+5. Run ```bundle install```.
+6. Run ```yarn install```.
+7. Run ```rails db:migrate```.
+8. Run ```rails server``` or ```rails s```.
+
+
+## Author Details:
+
+👤 **Sinan Sevgi**
+
+- GitHub: [@sinansevgi](https://github.com/sinansevgi)
+- LinkedIn: [@sinansevgi](https://www.linkedin.com/in/sinan-s-52559437/)
+- Twitter: [@nativeofcybers1](https://twitter.com/nativeofcybers1)
+
+
+👤 **Chad-Leigh Van Wyk**
+
+- Github:[@El-Potato-Slayer](https://github.com/El-Potato-Slayer)
+- Twitter:[@elpotatoslayer](https://twitter.com/elpotatoslayer)
+- LinkedIn: [Chad van Wyk](https://www.linkedin.com/in/chad-van-wyk-4228b21a6/)
+
+## Acknowledgment
+The resources we have used to develop our project:
+
+- [Odin-project](https://www.theodinproject.com/courses/ruby-on-rails/lessons/building-with-active-record-ruby-on-rails)
+- [Tutorialspoint](https://www.tutorialspoint.com/ruby-on-rails/rails-migrations.htm)
+- [Guides](https://guides.rubyonrails.org/active_record_validations.html#length)
+
+## Show your support
+
+Give ⭐ to me on GitHub — it helps!
+
+## 📝 License
+
+This project is [MIT](lic.url) licensed.  
```

Comment:
The license link is broken. Kindly fix it.

Reply:
gg